README.md

```diff
@@ -17,6 +17,15 @@ Maintainer: Joakim Gross <joakim.gross@pelagicore.com>
 * texlive-latex-extra (when building PDF)
 * texlive-latex-recommended (when building PDF)
 
+###  Install build dependencies on Debian
+
+```
+sudo apt-get install python-pip
+pip install sphinxcontrib-seqdiag sphinxcontrib-blockdiag \
+              sphinxcontrib-actdiag sphinxcontrib-actdiag \
+              sphinxcontrib-manpage
```

Comment:
I think these packages are available only through pip. 

I tried `ap-get`-ing them and it didn't work. Maybe @breakreturn knows the details.